services/httpoverrpc/client/client.go

```diff
@@ -226,16 +227,20 @@ type getCmd struct {
 	protocol            string
 	hostname            string
 	insecureSkipVerify  bool
+	dialAddress         string
 }
 
 func (*getCmd) Name() string     { return "get" }
-func (*getCmd) Synopsis() string { return "Makes a HTTP call to a port on a remote host" }
+func (*getCmd) Synopsis() string { return "Makes a HTTP(-S) call to a port on a remote host" }
 func (*getCmd) Usage() string {
-	return `get [-method METHOD] [-header Header...] [-body body] [-protocol Protocol] [-hostname Hostname] remoteport request_uri:
+	return `get [-method METHOD] [-header Header...] [-dial-address dialAddress] [-body body] [-protocol Protocol] [-hostname Hostname] remoteport request_uri:
   Make a HTTP request to a specified port on the remote host.
 
-  Example:
+  Examples:
+  # send get request to https://localhost:9090/hello
   httpoverrpc get --hostname 10.1.23.4 --protocol https 9090 /hello
```

Comment:
nit: It looks like the first example is sending a request to `10.1.23.4:9090/hello`, not `localhost:9090/hello`